js/page_state.js

```diff
@@ -0,0 +1,18 @@
+import {formElement,FORM_FIELD,filterElement,FILTER_FIELD} from './data.js';
+import {changeAttributeDisabled} from './util.js';
+
+const setActive = () => {
+  formElement.classList.remove('ad-form--disabled');
+  filterElement.classList.remove('ad-form--disabled');
+  changeAttributeDisabled(FILTER_FIELD, false);
+  changeAttributeDisabled(FORM_FIELD, false);
+};
+
+const setInactive = () => {
+  formElement.classList.add('ad-form--disabled');
+  filterElement.classList.add('ad-form--disabled');
+  changeAttributeDisabled(FILTER_FIELD, true);
+  changeAttributeDisabled(FORM_FIELD, true);
+};
```

Comment:
как видно, код функций повторяется, а значит можно объединить в одну функцию вроде такой:

```suggestion
const togglePageActiveState = (bool) => {
  formElement.classList.toggle('ad-form--disabled', bool); // метод toggle умеет принимать второй аргумент. Это удобно как раз для таких случаев
  filterElement.classList.toggle('ad-form--disabled', bool);
  toggleDisabledOnFormNodes(bool); // можно объединить в одну функцию учитывая, что обе формы у нас отключаются/включаются одновременно
};
```